hawkbit-rest/hawkbit-rest-docs/src/main/asciidoc/rootcontroller-api-guide.adoc

```diff
@@ -301,6 +305,64 @@ include::../errors/409.adoc[]
 include::../errors/415.adoc[]
 include::../errors/429.adoc[]
 |===
+
+== GET /{tenant}/controller/v1/{controllerid}/installedBase/{actionId}
+
+=== Implementation notes
+
+Resource to receive information of the latest installation.
+Can be used to re-retrieve artifacts of the last finished action, for example in case a re-installation is necessary.
+The response will be of the same format as the deploymentBase operation, providing the latest action that has been finished successfully.
+As the action is already finished, no further feedback is expected.
```

Comment:
As accessing not only the latest but all previous installed actions is possible maybe replace `latest` with `previous`.

Reply:
Right, that wording makes it clearer that the endpoint can be called for any older installed action. 
Docu is adapted.